### PR TITLE
core: Add support for fake signatures, enabled per app by dynamic permission

### PR DESCRIPTION
**Pull request by dic1911**
Original Commit here: https://gerrit.omnirom.org/#/c/16080/

Change-Id: I456650338399abcbc22f97fa09fe50eb648b9e2e

### CODE REVIEW

File: core/res/res/values/xosp_strings.xml [hunk 1]
```diff
@@ -25,4 +25,13 @@
     <!-- Restart Progress Dialog. This is shown if the user chooses to restart the phone. -->
     <string name="reboot_progress">Restarting\u2026</string>
 
+    <!-- Button to soft reboot the device, within the Reboot Options dialog -->
+   <string name="reboot_systemui">Restart SystemUI</string>
```
Comment (nilac8991) [inline core/res/res/values/xosp_strings.xml]: Un used string

Comment (dic1911) [inline core/res/res/values/xosp_strings.xml]: Strange... I didn't modify these 2 lines

Comment (nilac8991) [inline core/res/res/values/xosp_strings.xml]: Just remove those extra strings and amend the commit

Comment (dic1911) [inline core/res/res/values/xosp_strings.xml]: Can't I just use the online editor to remove those 2 lines?

**Pull request merged**